### ISSUE TITLE
Duplicating system data graphs

### ISSUE THREAD

**Issue by vol1ura** [igorkasyanchuk/rails_performance#118]
Having rails and sidekiq processes on the same server, CPU and Storage charts shows the same things

**Comment by vol1ura** [comment]
yes, here are 2 sections but cpu and storage graphs seems to be the same

![image](https://github.com/user-attachments/assets/d2fb9976-5b0a-4c3c-b741-c6f58e21d09c)

**Comment by igorkasyanchuk** [comment]
I would leave it for now, maybe later I'll group data just by server ID

**Comment by vol1ura** [comment]
yes, I think cpu and storage should be grouped by server ID, and then go memory charts

**Comment by igorkasyanchuk** [comment]
Can you try to implement something like this? Each line with own color.

ideally we should also have controls on UI to hide/show different lines on the chart

**Comment by vol1ura** [comment]
ok, maybe later
